docs/modules/usage/about.md

```diff
@@ -9,41 +9,19 @@ Achieving full replication of production-grade applications with LLMs is a compl
 
-## 🤝 How to Contribute
-
-OpenHands is a community-driven project, and we welcome contributions from everyone. Whether you're a developer, a researcher, or simply enthusiastic about advancing the field of software engineering with AI, there are many ways to get involved:
-
-- **Code Contributions:** Help us develop the core functionalities, frontend interface, or sandboxing solutions
-- **Research and Evaluation:** Contribute to our understanding of LLMs in software engineering, participate in evaluating the models, or suggest improvements
-- **Feedback and Testing:** Use the OpenHands toolset, report bugs, suggest features, or provide feedback on usability
-
-For details, please check [this document](https://github.com/All-Hands-AI/OpenHands/blob/main/CONTRIBUTING.md).
-
-## 🤖 Join Our Community
-
-We have both Slack workspace for the collaboration on building OpenHands and Discord server for discussion about anything related, e.g., this project, LLM, agent, etc.
-
-- [Slack workspace](https://join.slack.com/t/opendevin/shared_invite/zt-2oikve2hu-UDxHeo8nsE69y6T7yFX_BA)
-- [Discord server](https://discord.gg/ESHStjSjD4)
-
-If you would love to contribute, feel free to join our community. Let's simplify software engineering together!
-
-🐚 **Code less, make more with OpenHands.**
-
-[![Star History Chart](https://api.star-history.com/svg?repos=All-Hands-AI/OpenHands&type=Date)](https://star-history.com/#All-Hands-AI/OpenHands&Date)
-
-## 🛠️ Built With
+## Built With
 
 OpenHands is built using a combination of powerful frameworks and libraries, providing a robust foundation for its development. Here are the key technologies used in the project:
 
 ![FastAPI](https://img.shields.io/badge/FastAPI-black?style=for-the-badge) ![uvicorn](https://img.shields.io/badge/uvicorn-black?style=for-the-badge) ![LiteLLM](https://img.shields.io/badge/LiteLLM-black?style=for-the-badge) ![Docker](https://img.shields.io/badge/Docker-black?style=for-the-badge) ![Ruff](https://img.shields.io/badge/Ruff-black?style=for-the-badge) ![MyPy](https://img.shields.io/badge/MyPy-black?style=for-the-badge) ![LlamaIndex](https://img.shields.io/badge/LlamaIndex-black?style=for-the-badge) ![React](https://img.shields.io/badge/React-black?style=for-the-badge)
 
 Please note that the selection of these technologies is in progress, and additional technologies may be added or existing ones may be removed as the project evolves. We strive to adopt the most suitable and efficient tools to enhance the capabilities of OpenHands.
 
-## 📜 License
+## Contributing, Community Servers, Licensing
 
-Distributed under the MIT License. See [our license](https://github.com/All-Hands-AI/OpenHands/blob/main/LICENSE) for more information.
+For guides on how to contribute to OpenHands, joining our Discord and Slack servers and licensing
+[check out the OpenHands README.](https://github.com/All-Hands-AI/OpenHands?tab=readme-ov-file#-how-to-contribute)
```

Comment:
Why do we change this? The change no longer displays a simple sentence about MIT license.

Reply:
Just dedup since it's here and referenced: https://github.com/All-Hands-AI/OpenHands/pull/5060

If it's important to you, I can add it back.

Reply:
I can also do something like:
See the README for Contributing to Openhands (link), Community Servers (link) and Licensing (link).

Let me know what you prefer.

Reply:
We are on the documentation site, right, not on GitHub? IMHO we can keep the license information visible, plain and simple: MIT license (link).

Reply:
How about "Distributed under MIT license (link). For guides on how to contribute to OpenHands, joining our Discord and Slack servers ..."

Reply:
@enyst updated per your suggestions. Thanks!

Reply:
Please give me a 👍 if you like what you see.

Reply:
Thank you! And sorry @mamoodi, I was typing in a hurry. One point here is that everyone knows what MIT License is. Understood at a single glance, you don't need to look for it. I would really prefer that. 😊 

Another point is that this is on all-hands site, where there are other things called "licensing" - and other licenses. IMHO, when it comes to this topic, what seems like duplication is good. It's actually differentiation.

Just keeping things clear and easy to see, what applies to what.

I'll take rotten tomatoes if you wanna throw at me, but big thank you for doing this! 😅

Reply:
hahaha all good! Sometimes I try to make things too simple.
Anything to make @enyst happy :)